apps/admin/backend/src/store.ts

```diff
@@ -2132,6 +2150,7 @@ export class Store {
         status: 'pending',
         isUnmarked: fromSqliteBool(row.isUnmarked),
         isUndetected: fromSqliteBool(row.isUndetected),
+        machineMarkedText: row.machineMarkedText,
```

Comment:
Previously this field was provided to the frontend via the `getWriteInImageAndLayout`, which doesn't make as much sense anymore with the ballot images no longer so tightly coupled to write-ins. Regardless, I think it makes more sense for `machineMarkedText` to be provided to the frontend on the `WriteInRecord` object, so it is now included with write-ins in `getWriteInRecords`.